conductor/src/main/java/com/bluelinelabs/conductor/Controller.java

```diff
@@ -92,6 +92,8 @@
     private boolean isPerformingExitTransition;
     private boolean isContextAvailable;
 
+    static volatile ControllerFactory FACTORY = new ControllerFactory.DefaultControllerFactory();
```

Comment:
I'm not sure if this is the best place for this to live. Maybe there is a better place for it? Does it need to be static? Would make sense for a `Router` to have a `ControllerFactory`?